packages/openid4vc/src/openid4vc-issuer/router/accessTokenEndpoint.ts

```diff
@@ -215,13 +269,15 @@ export function handleTokenRequest(config: OpenId4VcIssuerModuleConfig) {
         clientId: issuanceSession.clientId,
 
         additionalAccessTokenPayload: {
-          'pre-authorized_code':
-            grant.grantType === preAuthorizedCodeGrantIdentifier ? grant.preAuthorizedCode : undefined,
+          'pre-authorized_code': issuanceSession.preAuthorizedCode,
           issuer_state: issuanceSession.authorization?.issuerState,
```

Comment:
❓ Can this change cause issues? Do we have any flow where you might have both set? My issue here was to know what to set in the case of refresh tokens.

Reply:
An offer can contain both

It is a change in behaviour, since we only add the used method to the token.

We can base it on the current access token? That is used for refresh